### ISSUE TITLE
EBS - Performance regression when using external capture card with m73v21

### ISSUE THREAD

**Issue by agouaillard-cosmo** [CoSMoSoftware/OBS-studio-webrtc#156]
- Using anything that produces HDMI output as a video device for capturing audio and video at 1080p@30fps
- an AJA U-TAP device for transforming the incoming HDMI audio and video streams into an ??? stream over USB (not critical nor blocking information)
- using a Mac os reference computer configuration with EBS to get the video and audio frames and stream them out to Evercast back-end as a WebRTC media stream.
- Using chrome, we connect to the same room EBS is streaming to in parallel.
- we suppose most settings beyond resolutions inconsequential at this stage, but will document it if they appear to be important subsequently.

Problem:
- using Evercast's m65v22 version, the CPU footprint of EBS is: ???
- using CoSMo's m73v21 version, the CPU footprint of EBS is: ???

Hypothesis: some performance improvement were made in OBS after v21. We shall test with m73v23.2 before investigating further.

**Comment by agouaillard-cosmo** [comment]
Seems to be fixed in m73v23.2
pending validation by brad.

**Comment by agouaillard-cosmo** [comment]
Mac Hardware
![Screen Shot 2019-10-05 at 10 18 59 PM](https://user-images.githubusercontent.com/27793645/66260375-332e8500-e7be-11e9-980f-6ec85378ad53.png)

Configuration:
Switch connected via HDMI (1080@30fps) to U-TAP HDMI.

separate Validation of  capture card in chrome and safari using getusermedia samples:
https://webrtc.github.io/samples/src/content/devices/input-output/

EBS's full CPU trace in the Activity Monitor:
- Version                 :  Capture-only / +VP8 / +VP9
- official m65v22.x :          30          /   105  / -
- new     m73v23.2 :          15           /     60  / 100

![Screen Shot 2019-10-05 at 10 24 19 PM](https://user-images.githubusercontent.com/27793645/66260696-5f4c0500-e7c2-11e9-9684-ffc3a40ba80c.png)
![Screen Shot 2019-10-05 at 10 24 03 PM](https://user-images.githubusercontent.com/27793645/66260697-5f4c0500-e7c2-11e9-9f7b-1d05c930876f.png)
![Screen Shot 2019-10-05 at 10 23 22 PM](https://user-images.githubusercontent.com/27793645/66260698-5f4c0500-e7c2-11e9-9739-0600ec3884d7.png)
![Screen Shot 2019-10-05 at 10 31 11 PM](https://user-images.githubusercontent.com/27793645/66260700-66731300-e7c2-11e9-9e6a-c7e706b07d71.png)
![Screen Shot 2019-10-05 at 10 30 06 PM](https://user-images.githubusercontent.com/27793645/66260701-66731300-e7c2-11e9-97f2-73d497536a4f.png)
![Screen Shot 2019-10-05 at 10 57 28 PM](https://user-images.githubusercontent.com/27793645/66260807-f8c7e680-e7c3-11e9-9c1d-1ddb5a5ded40.png)

**Comment by agouaillard-cosmo** [comment]
Hi Damien.
Just to confirm this is also with the Capture device as input, or this is with screen capture (in which case, we should make a separate ticket)?
